src/requests.js

```diff
@@ -11,16 +11,29 @@ export const getCategories = () => {
         .get('/categories')
         .then((res) => {
             return res;
-    })
-}
+        })
+};
 
-export const getReviews = (category) => {
+export const getReviewById = (review_id) => {
+    return gamesApi
+        .get(`/reviews?review_id=${review_id}`)
+        .then((res) => {
+            return res.data.reviews[0];
+        })
+};
+
+export const getReviews = (queryObject) => {
 
-    let queryParams = category && category.slug ? `?category='${category.slug}'` : '';
+    let queryStr = '';
+
+    if (queryObject && queryObject.category) queryStr += `?category='${queryObject.category}'`;
+    if (queryObject && queryObject.review_id) queryStr += `?review_id=${queryObject.review_id}`;
         
     return gamesApi
-        .get(`/reviews${queryParams}`)
+        .get(`/reviews${queryStr}`)
```

Comment:
You can refactor this to:

`.get(`/reviews`, {params: {review_id, category}})`

This will allow you to easily scale your project up to allow sort/order queries as well :)